blocks/sticky-nav/sticky-nav.css

```diff
@@ -0,0 +1,108 @@
+div.sticky-nav ul li a {
```

Comment:
Removing the divs would be beneficial since this restriction adds no value but restricts the flexibility of changing the parent tag if necessary.
It will also be less cluttered.

Reply:
Fixed